### PR TITLE
Update README.md to account for VISION.md

### PR DESCRIPTION
**Pull request by jeddy3**
Highlights the _three_ types of rules, both in the list of features and within the getting started section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,22 @@
...
+-   **Support for the latest CSS syntax:** Including custom properties, calc() and nesting.
 -   **Understands *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it understands any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and *experimental support* for Less.
 -   **Completely unopinionated:** Only enable the rules you want, and configure them with options that tailor the linter to your needs.
 -   **Support for plugins:** It's easy to create your own rules and add them to the linter.
 -   **Automatically fixes some stylistic warnings:** Save time by having stylelint fix your code with this *experimental* feature.
 -   **Shareable configs:** If you don't want to craft your own config, you can extend a shareable config.
 -   **Options validator:** So that you can be confident that your config is valid.
--   **Well tested:** Nearly twenty five thousand tests cover the internals and rules.
+-   **Well tested:** Over ten thousand tests cover the internals and rules.
```
Comment (ntwb) [inline README.md]: Oh! I didn't realise the number of tests removed was _this_ many

**Comment by ntwb** [comment]
p.s. @jeddy3 Can you "pin" the https://github.com/stylelint/stylelint-config-recommended repo to https://github.com/stylelint/stylelint so that it shows at the top of the GitHub org repo list please

File: README.md (hunk 1 above)
Comment (jeddy3) [inline README.md]: I think the way the tests were counted by tape is different to how they are counted now by jest.

**Comment by jeddy3** [comment]
> Can you "pin" the https://github.com/stylelint/stylelint-config-recommended repo to https://github.com/stylelint/stylelint so that it shows at the top of the GitHub org repo list please

Done

**Pull request merged**